src/generics.rst

```diff
@@ -48,21 +48,13 @@ Generic Parameters
 
 :dp:`fls_sye3d17l9bf5`
 A :t:`generic parameter` is a placeholder for a :t:`constant`, a :t:`lifetime`,
-or a :t:`type` whose :t:`value` is supplied statically by a
-:t:`generic argument`.
-
-:dp:`fls_4a2qshaf5se7`
-It is a static error to use a :t:`generic parameter` in the
-:t:`discriminant initializer` of an :t:`enum variant`.
+or a :t:`type` whose :t:`constant`, :t:`lifetime`, or :t:`type` is supplied
```

Comment:
This is a bit confusing, it looks like the type has a constant, a lifetime, or a type. Why doesn't value work here?

Reply:
A value is an instance of a type. What one is supplying to a lifetime parameter is a lifetime though, for a type parameter you supply a type, and for a constant parameter you supply a constant (value). So value is not the correct term here

Reply:
Good point! See suggestion.